### DISCUSSION TITLE
Access control [Apr 30]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5611] [Discord Threads]
> 🙋‍♂️ **Santawong**

**Comment by o1lab** [comment]
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### You can share only selected table & view with a "role" using UI ACL; this currently is not possible at individual collaborator level though   
####    
#### https://docs.nocodb.com/setup-and-usages/meta-management#ui-access-control  
#  
> 🙋‍♂️ **Santawong**  
#### so i can only hide the table so people dont have access to the table but i cant stopping them for say editing a field that is linked from the table i have hide  
#  
> 🙋‍♂️ **Santawong**  
#### is my understanding correct ?  
#  
> 🙋‍♂️ **Santawong**  
#### thank you so much for the link btw it was a big help already  
#  
> 🙋‍♂️ **Santawong**  
#### I can't also have many different editor with different permission for a table .  
#  
> 🙋‍♂️ **Dstala | NocoDB Team**  
#### - you can provide access to specific view (with access to limited columns). but you won't be able to configure some columns as read only & some as read write    
#### - you are right. you can't stop them from editing a linked field when you provide access to a table   
#### - you can't have different editor with different permissions for a table  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1091186377695174867)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
